test/config-next/crl-storer.json

```diff
@@ -23,7 +23,8 @@
 		"issuerCerts": [
```

Comment:
Since inactive issuers can sign CRLs, OCSP, and final certs, should `int-ecdsa-c` and `int-rsa-c` also be added to these configs?